api/graphql/schema.graphql

```diff
@@ -634,6 +634,7 @@ type ApplicantFilter {
   expectedClassifications: [Classification]  # Filters applicants based on the classifications they've expressed interest in.
     @belongsToMany(relation: "classifications")
   skills: [Skill] @belongsToMany
+  # request creation connects to qualifiedClassifications
```

Comment:
Solve a point of confusion for myself

Reply:
I'm not sure what this comment means 😅

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/6897#issuecomment-1589752697